manual/en-US/coding-standards/chapters/docblocks.md

```diff
@@ -98,6 +98,7 @@ Function definitions must include a documentation comment in accordance with the
 * Short description (required, followed by a blank line)
 * Long description (optional, followed by a blank line)
 * @param (required if there are method or function arguments, the last @param tag is followed by a blank line)
+* @throws (required if method or function arguments throw a specific type of exception, the last @throws tag is followed by a blank line)
```

Comment:
`@throws` tags are placed after the `@since` tag.

https://github.com/joomla/joomla-cms/blob/staging/libraries/cms/application/cms.php#L153